ListOfQuestions.md

```diff
@@ -23,4 +23,6 @@ Week1:
 * pattern match as in scala:
 case xs :: xxs =>
 
-* check emptyness in function args.
\ No newline at end of file
+* check emptyness in function args.
+
+* cons vs conj
```

Comment:
cons

* first an element, then the collection
* always returns a linked list segment, consisting of first and rest.
* prepends elements to the front

conj

* first collection, then element(s) to insert
* retains the original collection type
* puts elements where it's efficient with the collection:
  * to the end for vectors
  * to the front for lists
  * just inserts elements into sets
  * inserts key-values into maps
  * can accept several elements at a time